### DISCUSSION TITLE
Make create option always visible, even when not writing. Any idea on how to implement?

### DISCUSSION THREAD

**Discussion by AndreaEsposit** [JedWatson/react-select#5778] [Q&A] [answered]
I am attempting to ensure that the 'Create 'X'' option remains visible at all times, even when the user hasn't entered any value. The goal is to provide a clear indication to the user that they can create. If achieving this isn't possible, how would you do it?

**Comment by manjushsh** [comment, marked as answer]
Hi @AndreaEsposit , it is possible to achieve this with `isValidNewOption` props.
You can check it on [this code sandbox ](https://codesandbox.io/s/creatable-select-always-creatable-wn9v25?file=/src/App.js:978-1021)

If you need 'Create "x" ' all time, even when empty, just pass `isValidNewOption={(input) => () => true}` which will show up like image attached below.

![image](https://github.com/JedWatson/react-select/assets/94426452/d996214c-759f-42fb-81d1-cedd518b95ca)

Sample Code to show Create text once user enters anything in input:

```
<CreatableSelect
        isClearable
        isValidNewOption={(input) => input?.length}    // For always show:   () => true     For always hide: () => false
        // menuIsOpen
        isDisabled={isLoading}
        isLoading={isLoading}
        onChange={(newValue) => setValue(newValue)}
        onCreateOption={handleCreate}
        value={value}
        options={options}
      />
```

**Reply by AndreaEsposit** [reply]
Thank you for the help :)
